### DISCUSSION TITLE
Your Mermaid Flowchart for content processing, where can i get the code? what does it look like? HELP

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#3] [Q&A] [answered]
# This is what it looks like:

![image](https://user-images.githubusercontent.com/29872822/98165126-3355f400-1e9a-11eb-87dc-dd10f958c7b1.png)

You can download my templates, CSS, and mermaid code after signing up for my newsletter through this link: 

[[Get My Templates and CSS]](https://bryanjenkstech.ck.page/d4ec0713d5)

**Comment by asdfgeoff** [comment]
Thanks for sharing @tallguyjenks!

A follow-up question: how did you make your nodes link to pages in Obsidian on click?

**Comment by tallguyjenks** [comment]
@asdfgeoff check #10 :)

**Comment by tallguyjenks** [comment, marked as answer]
Converted from issue
